superset/templates/appbuilder/navbar_right.html

```diff
@@ -87,3 +87,19 @@
     <li><a href="{{appbuilder.get_url_for_login}}">
     <i class="fa fa-fw fa-sign-in"></i>{{_("Login")}}</a></li>
 {% endif %}
+
+<li>
+  <a href="/static/assets/version_info.json" title="Version info">
```

Comment:
Would be better to show the actual version number than linking to json

Reply:
Not a bad idea. I can look into doing that. My goal here was mostly to revert the removal, but I agree that being able to see the version by hovering is better than opening a link that shows JSON.

Reply:
The `fork` icon is also probably not the best icon here.

Reply:
How about `info`? 
<img width="26" alt="Screen Shot 2019-04-16 at 9 28 50 AM" src="https://user-images.githubusercontent.com/8096320/56213458-0f1d9280-602a-11e9-978e-36bfe0412a50.png">